NuGet.config

```diff
@@ -2,6 +2,9 @@
 <configuration>
   <packageSources>
     <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
+    <!--<add key="AspNetVNext" value="https://www.myget.org/F/aspnetvnext/api/v3/index.json" />-->
+    <add key="XunitMyget" value="https://www.myget.org/F/xunit/api/v3/index.json" />
```

Comment:
Can we merge this to develop as opposed to master? Develop can later be merged to master, as we will not have myget feeds going out in "nuget.org" (which reflects the content in master).